### PR TITLE
Change Typeform answers payload

### PR DESCRIPTION
**Pull request by monsha**
Now answers look like this:
```
          "answers": {
            "opinion_scale_58048690": 3,
            "long_text_58049969": "It is really helpful to see your credit score at any time and if it changes then why gas it changed",
            "opinion_scale_58048493": 5,
            "opinion_scale_58048704": 2,
            "dropdown_58049487": "South East",
            "opinion_scale_58048049": 10,
            "opinion_scale_58048684": 3,
            "opinion_scale_58048687": 3,
            "multiple_choice_58049393": {
              "label": "55 to 64"
            },
            "dropdown_58049765": "< £20,000"
          }
```

### CODE REVIEW

File: app/models/agents/typeform_response_agent.rb [hunk 1]
```diff
@@ -31,58 +31,59 @@ class TypeformResponseAgent < Agent
...
+            "opinion_scale_58048690": 3,
+            "long_text_58049969": "It is really helpful to see your credit score at any time and if it changes then why gas it changed",
+            "opinion_scale_58048493": 5,
+            "opinion_scale_58048704": 2,
+            "dropdown_58049487": "South East",
+            "opinion_scale_58048049": 10,
+            "opinion_scale_58048684": 3,
+            "opinion_scale_58048687": 3,
+            "multiple_choice_58049393": {
+              "label": "55 to 64"
```
Comment (duboff) [inline app/models/agents/typeform_response_agent.rb:120]: Why does this one have a different format?

**Comment by duboff** [comment]
@monsha to be safe, can we save this in a new field, not overwrite answers?

**Review by OpakAlex — APPROVED** [review]

File: app/models/agents/typeform_response_agent.rb (hunk 1 above)
Comment (monsha) [reply app/models/agents/typeform_response_agent.rb:120]: Each tuple is an answer, the key is the combination between field.type and value (text, number, choice..)

**Comment by monsha** [comment]
@duboff Ok, I will add another field

**Comment by monsha** [comment]
@dubbof changes applied!

**Review by duboff — APPROVED** [review]
👍  Niiice. Please deploy

**Pull request merged**